daffodil-core/src/main/scala/org/apache/daffodil/dsom/RawCommonRuntimeValuedPropertiesMixin.scala

```diff
@@ -21,25 +21,25 @@ import org.apache.daffodil.schema.annotation.props.PropertyMixin
 
 trait RawCommonRuntimeValuedPropertiesMixin
   extends PropertyMixin {
-  protected final lazy val optionByteOrderRaw = findPropertyOption("byteOrder")
+  protected final lazy val optionByteOrderRaw = findPropertyOption("byteOrder", expressionAllowed = true)
   protected final lazy val byteOrderRaw = requireProperty(optionByteOrderRaw)
-  protected final lazy val optionEncodingRaw = findPropertyOption("encoding")
+  protected final lazy val optionEncodingRaw = findPropertyOption("encoding", expressionAllowed = true)
   protected final lazy val encodingRaw = requireProperty(optionEncodingRaw)
-  protected final lazy val optionOutputNewLineRaw = findPropertyOption("outputNewLine")
+  protected final lazy val optionOutputNewLineRaw = findPropertyOption("outputNewLine", expressionAllowed = true)
   protected final lazy val outputNewLineRaw = requireProperty(optionOutputNewLineRaw)
 
-  protected final lazy val optionFillByteRaw = findPropertyOption("fillByte")
+  protected final lazy val optionFillByteRaw = findPropertyOption("fillByte", expressionAllowed = true)
```

Comment:
fillByte is a DFDL String Literal and so cannot be an expression:

https://daffodil.apache.org/docs/dfdl/#_Toc130873645